includes/class-micropub-endpoint.php

```diff
@@ -49,8 +49,18 @@ public static function return_micropub_error( $response, $handler, $request ) {
 		return $response;
 	}
 
-
-
+	public static function log_error( $message, $name = '' ) {
+		if ( empty( $message ) ) {
+			return false;
+		}
+		if ( is_array( $message ) || is_object( $message ) ) {
+			$message = wp_json_encode( $message );
+		}
+		if ( ! is_string( $name ) || empty( $name ) ) {
+			$name = 'Micropub: ';
```

Comment:
Replace `$name = 'Micropub: ';` with `$name = 'Micropub';` becaue the `: ` is set by the error logger in line 62